### PR TITLE
[GRD-80961,GRD-80162 and GRD-63127] Azure Postgres Publish new fixes

### PR DESCRIPTION
**Pull request by apurva-birajdar**
1. Updating conf
2. Updating ReadMe
3. Updating source code and zip

### CODE REVIEW

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 1]
```diff
@@ -1,266 +1,188 @@
 # Azure PostgreSQL-Guardium Logstash filter plug-in
-### Meet Azure PostgreSQL
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:2]: Why did you remove the Meet Azure part?

Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:2]: added

**Comment by ofer-haim** [comment]
https://github.com/IBM/universal-connectors/tree/main/filter-plugin/logstash-filter-azure-postgresql-guardium#meet-azure-postgresql

**Review by ofer-haim — APPROVED** [review]

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 2]
```diff
@@ -4,263 +4,192 @@
...
-    
-12.	After successful creation of resource group, select resource button.
-    	
-13.	From settings,select connection security.
-
-14.	In firewall rules,select allow access to azure services to 'Yes'.
-
-15. Click on add current client IP address, and then click on Save.
-## Enabling Auditing
+2. Click the search bar.
```
Comment (ShalakaKulkarni15) [inline filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:30]: As discussed, check if these steps are required.

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 3]
```diff
@@ -4,263 +4,192 @@
...
-       •	log_line_prefix = specify as per requirement but should include timestamp,client ip,client port,database 
-   username,database name,process id application name,sql state. Refer from this link https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-LINE-PREFIX 
-							(eg:- %t:%r:%u@%d:[%p]:%a:%e)
-       
-       •	pgaudit.log = DDL,FUNCTION,READ,WRITE,ROLE
-   
-       •	pgaudit.log_catalog = off
-			
-       •	pgaudit.log_client = off
+      1. In **Settings**, select **server parameters** and set the server parameters as follows:
```
Comment (ShalakaKulkarni15) [inline filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:57]: Mentioned page name on which uses can see the settings options

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 4]
```diff
@@ -4,263 +4,192 @@
...
+         * log_checkpoints = off
+         * log_error_verbosity = VERBOSE 
+         * log_line_prefix = specify as per requirement but should include 
+         timestamp, client ip, client port, database username, database name, process id, application name,sql state.
```
Comment (ShalakaKulkarni15) [inline filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:61]: For information regarding timestamp, client ip, client port, database username, database name, process id, application name,sql state parameters, see [Error Reporting and Logging](https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-LINE-PREFIX).

**Review by ShalakaKulkarni15 — CHANGES_REQUESTED** [review]

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 5]
```diff
@@ -4,263 +4,192 @@
...
-	3. Log Analytics Workspace
-	4. Azure Partner Solution
+You can retrieve Azure PostgreSQL audit data in the following ways:
+1. Azure Event Hub
+2. Azure Storage
+3. Log Analytics Workspace
+4. Azure Partner Solution
 
-In this plugin we have used Azure Event Hub.
+For this plug-in, we used Azure Event Hub.
```
[the remark this answers is not in the document]
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:25]: done

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md (hunk 2 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:30]: As per your request removing first point where 'Procedure for DB creations' is mentioned to make it  compatible with other plugin's readMe.

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md (hunk 3 above)
Comment (apurva-birajdar) [reply filter-plugin/logstash-filter-azure-postgresql-guardium/README.md]: done

**Review by ShalakaKulkarni15 — APPROVED** [review]

**Pull request merged**